tests/UI/campaigns/functional/API/02_endpoints/02_attribute/04_postAttributesAttribute.ts

```diff
@@ -103,7 +129,100 @@ describe('API : POST /admin-api/attributes/attributes', async () => {
+
+      const textColumn = await boAttributesPage.getTextColumn(page, 1, 'name');
+      expect(textColumn).to.contains(dataAttributes.paperType.name);
+    });
+
+    it('should view attribute group', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'viewAttributeGroup', baseContext);
+
+      await boAttributesPage.viewAttribute(page, 1);
+
+      const pageTitle = await boAttributesViewPage.getPageTitle(page);
+      expect(pageTitle).to.equal(boAttributesViewPage.pageTitle(dataAttributes.paperType.name));
+    });
+
+    it('should reset all filters and get the id of last attribute value', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'resetFilterLast', baseContext);
+
+      numberOfAttributes = await boAttributesViewPage.resetAndGetNumberOfLines(page);
+      expect(numberOfAttributes).to.be.above(0);
+
+      const attributeId = parseInt(await boAttributesViewPage.getTextColumn(page, numberOfAttributes, 'id_attribute'), 10);
+      expect(attributeId).to.be.gt(0);
+      expect(attributeId).to.be.equal(jsonResponse.attributeId);
+
+      const attributeNameEN = await boAttributesViewPage.getTextColumn(page, numberOfAttributes, 'name');
+      expect(attributeNameEN).to.be.a('string');
+      expect(attributeNameEN).to.be.equal(jsonResponse.names[dataLanguages.english.locale]);
+    });
+
+    it('should go to edit value page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToEditValuePage', baseContext);
+
+      await boAttributesViewPage.goToEditValuePage(page, numberOfAttributes);
+
+      const pageTitle = await boAttributesValueCreatePage.getPageTitle(page);
+      const attributeNameEN = jsonResponse.names[dataLanguages.english.locale];
+      expect(pageTitle).to.contains(boAttributesValueCreatePage.editPageTitle(attributeNameEN));
+
+      const attributeColor = await boAttributesValueCreatePage.getInputValue(page, 'color');
+      expect(attributeColor).to.be.a('string');
+      expect(attributeColor).to.be.equal(jsonResponse.color);
+    });
   });
 
-  describe.skip('BackOffice : Delete the Attribute', async () => {});
+  describe('BackOffice : Delete the Attribute value', async () => {
+    it('should go to \'Catalog > Attributes & Features\' page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'returnToAttributesPage', baseContext);
+
+      await boDashboardPage.goToSubMenu(
+        page,
+        boDashboardPage.catalogParentLink,
+        boDashboardPage.attributesAndFeaturesLink,
+      );
+      await boAttributesPage.closeSfToolBar(page);
+
+      const pageTitle = await boAttributesPage.getPageTitle(page);
+      expect(pageTitle).to.contains(boAttributesPage.pageTitle);
+    });
+
+    it('should view attribute group', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'viewAttributeGroup', baseContext);
```

Comment:
```suggestion
      await testContext.addContextItem(this, 'testIdentifier', 'viewAttributeGroupAfterDelete', baseContext);
```

Reply:
Ah I thought about the same modif alone ^^